frontend/src/components/LandingBoxes.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useState } from "react";
+import { useAuth } from "../context/AuthContext";
+import { Game, GamesServiceResponse, fetchGamesServiceAPI } from "../utilities";
+
+export default function LandingBoxes() {
+	const [games, setGames] = useState<Game[]>([]);
+	const { getAuthToken } = useAuth();
+
+	useEffect(() => {
+		const update = async () => {
+			try {
+				const gamesServiceData: GamesServiceResponse | null =
```

Comment:
You ok to remove this fetch entirely and just use the potentially stale data from `const { getUser } = useAuth()` so that refreshing the page doesn't kill our API quota like we did earlier? This will also make loading the landing page much faster for already signed in users.

Then the only way a user can get new data is from signing out and back in or refreshing the search bar